app/helpers/ems_container_helper/textual_summary.rb

```diff
@@ -131,9 +131,9 @@ def textual_volumes
   def textual_endpoints
     return unless @ems.connection_configurations.hawkular
 
-    [{:label => N_('Hawkular Host Name'),
+    [{:label => _('Hawkular Host Name'),
```

Comment:
@mzazrivec your previous commits used " instead of ', to maintain consistency, can you switch those to "